testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/UserTest.java

```diff
@@ -630,6 +630,24 @@ private List<String> createUsers() {
         return ids;
     }
 
+    @Test
+    public void countByAttribute() {
```

Comment:
Can you please add also a scenario where no user matches the count query?

Reply:
Hey @mhajas 
Could you pointing me where are release notes?

Reply:
Done